README.md

```diff
@@ -31,6 +31,9 @@ see [DNS Leakage](#dns-leakage) below.
 If you are using a distribution of Linux with uses the Arch User Repository, the
 simplest way to install is by using the [openvpn-update-systemd-resolved][aur]
 AUR package as this will take care of any updates through your package manager.
+[Debian](https://packages.debian.org/openvpn-systemd-resolved) and
+[Ubuntu](packages.ubuntu.com/openvpn-systemd-resolved) also provides a `.deb`
```

Comment:
Nit: could you please include the scheme in this URL, for consistency with other links?